istioctl/cmd/remove-from-mesh.go

```diff
@@ -124,12 +130,14 @@ func svcUnMeshifyCmd() *cobra.Command {
 			if len(args) != 1 {
 				return fmt.Errorf("expecting service name")
 			}
+			ns := handlers.HandleNamespace(namespace, defaultNamespace)
+			if analyzer_util.IsSystemNamespace(resource.Namespace(ns)) || ns == constants.IstioSystemNamespace {
```

Comment:
Can you change the fixed `constants.IstioSystemNamespace`  to the value passed via `-i` to exclude istio's system namespace?

Reply:
Done. Thanks.